src/main/groovy/liquibase/harness/BasicCompatibilitySuite.groovy

```diff
@@ -0,0 +1,10 @@
+package liquibase.harness
+
+import liquibase.harness.compatibility.basic.BasicCompatibilityTest
+import org.junit.runner.RunWith
+import org.junit.runners.Suite
+
+@RunWith(Suite.class)
+@Suite.SuiteClasses([BasicCompatibilityTest])
```

Comment:
Can we run these suites from terminal? I think we also need class in `src/test` to be able to run those, or different config in pom that will compile and run tests from `src/main`

Reply:
++